src/components/Faq.js

```diff
@@ -0,0 +1,41 @@
+import React, { useEffect } from "react";
+import { questions } from "./data2.js";
```

Comment:
We ask different questions on our platform, so this could have been a bit more descriptive to indicate it's for the FAQ